commit 23af890b3f04e80da783ba64e6b6d94822e01d54
Author: Ondřej Hošek <ondra.hosek@gmail.com>
Date:   Wed Aug 26 04:26:50 2020 +0200

    x86-64: Fix FMA4 detection in ifunc [BZ #26534]
    
    A typo in commit 107e6a3c2212ba7a3a4ec7cae8d82d73f7c95d0b causes the
    FMA4 code path to be taken on systems that support FMA, even if they do
    not support FMA4. Fix this to detect FMA4.

diff --git a/sysdeps/x86_64/fpu/multiarch/ifunc-fma4.h b/sysdeps/x86_64/fpu/multiarch/ifunc-fma4.h
index 76c677198dac5cb0..6cb70ce25209ee15 100644
--- a/sysdeps/x86_64/fpu/multiarch/ifunc-fma4.h
+++ b/sysdeps/x86_64/fpu/multiarch/ifunc-fma4.h
@@ -32,7 +32,7 @@ IFUNC_SELECTOR (void)
       && CPU_FEATURE_USABLE_P (cpu_features, AVX2))
     return OPTIMIZE (fma);
 
-  if (CPU_FEATURE_USABLE_P (cpu_features, FMA))
+  if (CPU_FEATURE_USABLE_P (cpu_features, FMA4))
     return OPTIMIZE (fma4);
 
   return OPTIMIZE (sse2);
